testapp/src/main/res/layout/window_first_time_miniapp.xml

```diff
@@ -0,0 +1,88 @@
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent">
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_alignParentStart="true"
+    android:orientation="vertical">
+
+    <RelativeLayout
+      android:id="@+id/firstTimeTopView"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:padding="@dimen/small_8"
+      android:background="@color/window_top_bar">
+
+      <ImageView
+        android:id="@+id/firstTimeCloseWindow"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:padding="@dimen/small_8"
+        android:src="@drawable/ic_close_24px" />
+
+      <TextView
+        android:id="@+id/firstTimeTutorial"
+        android:textStyle="bold"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
+        android:padding="@dimen/small_8"
+        android:text="@string/miniapp_first_time_tutorial"
+        android:textSize="@dimen/text_large_16" />
+
+    </RelativeLayout>
+
+    <ImageView
+      android:id="@+id/firstTimeAppIcon"
+      android:layout_width="@dimen/miniapp_icon_size"
+      android:layout_height="@dimen/miniapp_icon_size"
+      android:layout_marginTop="@dimen/small_8"
+      android:layout_marginStart="@dimen/small_8"
+      android:layout_gravity="center_horizontal"
+      android:src="@drawable/ic_default" />
+
+    <TextView
+      android:id="@+id/firstTimeMiniAppInfo"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="@dimen/small_8"
+      android:layout_gravity="center_horizontal"
+      android:paddingStart="@dimen/medium_16"
+      android:paddingEnd="@dimen/medium_16"
+      tools:text="Dummy MiniApp Info" />
+  </LinearLayout>
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_alignParentBottom="true"
+    android:padding="@dimen/small_8"
+    android:gravity="center_horizontal"
+    android:orientation="vertical">
+
+    <Button
+      android:id="@+id/firstTimeAccept"
+      style="?android:attr/borderlessButtonStyle"
+      android:textStyle="bold"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:background="@color/colorAccent"
+      android:text="@string/action_first_time_accept"
+      android:textColor="@android:color/white" />
+
+    <Button
+      android:id="@+id/firstTimeCancel"
+      android:textStyle="bold"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="@dimen/small_8"
+      android:layout_marginStart="@dimen/large_32"
```

Comment:
For me, it's a little bit strange looking that the "Cancel" button is slightly smaller than the "Accept" button. Maybe we could do this for the "Cancel" button instead:
- Remove the background color and shadow
- Make the text color red

So basically the button would just look like red text. What do you think?

Reply:
Sure, I have updated this part.